pkg/addon/quota.go

```diff
@@ -379,10 +379,85 @@ var quotaConfig string = `
 	}
 },
 {
-	"name": "0",
+	"name": "0E",
```

Comment:
curious what is this?  seems to be the same as the 100K

Reply:
E for evaluation. It's possible it can be removed. If the user is using an eval cluster they only get the option for the eval quota which has a value of 0 atm.